<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabDeveloperObject</class>
 <widget class="QWidget" name="TabDeveloperObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>330</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Cantarell</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>330</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDAC">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>330</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cantarell</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>DACs</string>
      </attribute>
      <layout class="QGridLayout" name="gridlayoutDac">
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="tabADC">
      <property name="font">
       <font>
        <family>Cantarell</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>ADCs</string>
      </attribute>
      <layout class="QGridLayout" name="gridlayoutAdc">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>350</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblReadout">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Readout: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QComboBox" name="comboDetector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select readout to read dac or adc info</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboDetector</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../include/icons.qrc"/>
 </resources>
 <connections/>
</ui>
